pinot-integration-tests/src/test/java/com/linkedin/pinot/integration/tests/QueryGenerator.java

```diff
@@ -171,6 +172,60 @@ public QueryGenerator(List<File> avroFiles, String pinotTableName, String h2Tabl
   }
 
   /**
+   * Helper method to store an Avro value into the valid SQL String value set.
```

Comment:
Avoid moving methods around, it makes it hard to see if they've been modified. Has there been any changes to this method?